pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -74,7 +76,7 @@ func NewAsyncAgentStore() AsyncAgentStore {
 
 // Store saves the value with a key to the storage, when there is a change in data
 // it fires up updated event
-func (s *AsyncAgentStore) Store(key AgentCommand, value interface{}) {
+func (s *AsyncAgentStore) Store(key, value any) {
```

Comment:
Why is it better to have the key as `any` instead of having it as concrete type?